### PR TITLE
feat: support load policy from text

### LINKED ISSUE #371: Support load policy text from adapter
> my model and policy are like this: [#367 (comment)](https://github.com/casbin/Casbin.NET/issues/367#issuecomment-2333998981) it gets passed in the string format itself. also, do we need the DefaultFileContentAdapter class now?

It need not now. we will add new API for it.

_Originally posted by @sagilio in https://github.com/casbin/Casbin.NET/issues/366#issuecomment-2365017684_

### PR DESCRIPTION
**Pull request by Taoyuesong**
close #371

This issue will be add new Text Adapter.

### CODE REVIEW

File: Casbin.UnitTests/ModelTests/EnforcerTest.cs [hunk 1]
```diff
@@ -700,28 +701,6 @@ public async Task TestInitEmptyAsync()
         await TestEnforceAsync(e, "alice", "/alice_data/resource1", "GET", true);
     }
 
-    [Fact]
-    public void TestInitEmptyByInputStream()
```
Comment (sagilio) [inline Casbin.UnitTests/ModelTests/EnforcerTest.cs:704]: Why remove this case, plz check

**Review by sagilio — CHANGES_REQUESTED** [review]
I believe it is not suitable to provide multiple modes in the FileAdapter, but rather to provide different Adapters. You can try to use the existing FileAdapter as a base class and extend it to create File, Stream, and Text three types of Adapters.

File: Casbin.UnitTests/ModelTests/EnforcerTest.cs (hunk 1 above)
Comment (Taoyuesong) [reply Casbin.UnitTests/ModelTests/EnforcerTest.cs:704]: It be moved to another file.

**Review by sagilio — APPROVED** [review]
LGTM

**Pull request merged**